docs/organizations/managing-people.md

```diff
@@ -19,7 +19,7 @@ To join or add an organization after having complete the signup process, click *
 
 **On Codacy Cloud**, organization owners can control who joins their organization by choosing one of the following options on the page **Plan and Billing** of the organization:
 
--   **Allow new people to join:** team members with access to the organization on the Git provider can join the organization on Codacy automatically. Your billing is updated accordingly.
+-   **Allow new people to join:** team members with access to the organization on the Git provider can join the organization on Codacy automatically. Your billing or remaining number of seats is updated accordingly.
```

Comment:
I don't understand why this is necessary. 

You are saying that new people will effect seats, that is an obvious consequence of new people 🤔

Reply:
Probably, the issue was with the existing sentence "Your billing is updated accordingly" that only addresses the case when customers pay monthly for their current number of seats. :thinking: 

@harrycochran, would you care to shed some light on this? Did you feel that the text wasn't explicit enough? Or perhaps this was asked by a customer?

Reply:
In the meantime, I made a proposal for an updated copy for these settings that I believe helps clarify when new users start being charged for:

https://docs.google.com/document/d/1YcnQTtBIGKG2U8gTOiEthu_TmdoBkDFYRxAYT2R7UlA/edit#heading=h.4m9a8hczc5tt

Reply:
@pauloribeiro-codacy & @inesgomas - sorry for the delay, I was out of office yesterday afternoon. So where this is coming from is we have a lot of cloud yearly customers & SH customers who have bought a package of seats (rather than having a CC linked). 

The current copy could be misinterpreted for these users because it says that we will update their billing, but really we are just taking this seat from their allowance/subscription. I think @pauloribeiro-codacy 's suggested update clears this up.